docs/career-and-open-positions/founding-developer-evangelist.md

```diff
@@ -10,18 +10,20 @@ We're backed by some of the world's [top investors](./#our-investors) and believ
 
 ## **Description**
 
-As our Founding Developer Evangelist, you will have two focus:
+As our Founding Developer Advocate, you will have three focus \(prioritized\):
 
-1. Generating awareness about Airbyte, by contributing and participating in guides, tutorials, webinars, meetups, conference panels, anything of value to the ecosystem
-2. Engaging with our users and contributors, so they become evangelists outside the company. 
+1. Helping us providing the best developer experience through our documentation and tutorials
+2. Engaging with our users and contributors, so they become evangelists outside the company.
```

Comment:
evangelist or advocate?

Reply:
Advocate, as that will be our main focus with them for a while. Evangelism is more about awareness, and I can help on that for a while.